src/doc/manifest.md

```diff
@@ -451,13 +451,19 @@ To structure your code after you've created the files and folders for your proje
 Files located under `examples` are example uses of the functionality provided by
 the library. When compiled, they are placed in the `target/examples` directory.
 
-They must compile as executables (with a `main()` function) and load in the
-library by using `extern crate <library-name>`. They are compiled when you run
+They may compile either as executables (with a `main()` function) or as libraries.
+They load in the library by using `extern crate <library-name>`. They are compiled when you run
 your tests to protect them from bitrotting.
 
-You can run individual examples with the command `cargo run --example
+You can run individual executable examples with the command `cargo run --example
 <example-name>`.
 
+Specify `crate-type` to make an example be compiled as a library:
+```toml
+[[example.example]]
```

Comment:
Ok well as the author of Cargo and implementor of this feature I'm saying `[[example.example]]` is invalid. The correct syntax is `[[example]]`